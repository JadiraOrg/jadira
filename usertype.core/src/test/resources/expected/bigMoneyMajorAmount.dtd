<!ELEMENT dataset (
    BIGMONEYMAJORAMOUNT*)>

<!ELEMENT BIGMONEYMAJORAMOUNT EMPTY>
<!ATTLIST BIGMONEYMAJORAMOUNT
    ID CDATA #REQUIRED
    BIGMONEY CDATA #IMPLIED
    NAME CDATA #IMPLIED
>

