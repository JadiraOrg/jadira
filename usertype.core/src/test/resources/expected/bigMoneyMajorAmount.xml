<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE dataset SYSTEM "bigMoneyMajorAmount.dtd">
<dataset>
  <BIGMONEYMAJORAMOUNT ID="0" BIGMONEY="100" NAME="test_0"/>
  <BIGMONEYMAJORAMOUNT ID="1" BIGMONEY="100" NAME="test_1"/>
  <BIGMONEYMAJORAMOUNT ID="2" BIGMONEY="0" NAME="test_2"/>
  <BIGMONEYMAJORAMOUNT ID="3" NAME="test_3"/>
</dataset>